### ISSUE TITLE
Omnisharp crashing / stops working and CodeLense does not display info

### ISSUE THREAD

**Issue by RononDex** [OmniSharp/omnisharp-roslyn#1550]
Still having issues with "latest" version:
CodeLense appears to not be working:
![image](https://user-images.githubusercontent.com/2249071/61043381-65566380-a3d6-11e9-9535-166bcfa1a910.png)
This is after having VSCode open untouched for like 1hour, so it should definitely have analyzed everything already.

It inserts the line where codelense would usually write the info, but there is no info being displayed

_Originally posted by @RononDex in https://github.com/OmniSharp/omnisharp-roslyn/issues/1549#issuecomment-510427015_

**Comment by filipw** [comment]
can you please provide the full OmniSharp log?

**Comment by RononDex** [comment]
So far the issue did not occur anymore, will update this issue with logs once it happens again

**Comment by RononDex** [comment]
So, after returning from my holiday and updating omnisharp / vscode, I now keep getting this error whenever I open the solution at work:
![image](https://user-images.githubusercontent.com/2249071/62453284-5616cb80-b772-11e9-83ae-f93f6ea5747a.png)

This crashes omnisharp and makes it unusable

**Comment by filipw** [comment]
can you share the repro? or at least provide a list of analyzer packages you use?

**Comment by RononDex** [comment]
I can not share the repo, its a private repo (non open source) and is hosted on azure devops

The analysers I use:
![image](https://user-images.githubusercontent.com/2249071/62457679-9cbcf380-b77b-11e9-9cef-d61e1f4aadee.png)

**Comment by filipw** [comment]
thanks, if you start an empty project, with the same analyzer set, does it work?

**Comment by RononDex** [comment]
Yes, I use the same analyers on my linux setup for private work and never had issues with them.
They also worked before my 3 weeks absence (haven't touched them since)

**Comment by filipw** [comment]
can you set `"omnisharp.logelevel": "trace"` post the full output of the omnisharp log?

**Comment by RononDex** [comment]
Just tried that, however it does not show more:
![image](https://user-images.githubusercontent.com/2249071/62460001-10adca80-b781-11e9-98ab-69ae4d97db5e.png)

Also it seems, the setting is unknown: 
![image](https://user-images.githubusercontent.com/2249071/62460024-1e635000-b781-11e9-890c-80aa73267291.png)

**Comment by filipw** [comment]
> EDIT: Just realized there is a "omnisharp.loggingLevel", trying that now

oops sorry 😂

**Comment by RononDex** [comment]
The last few entries:

```
[dbug]: OmniSharp.Stdio.Host
        ************ Request ************
{
  "Type": "request",
  "Seq": 220,
  "Command": "/v2/getcodeactions",
  "Arguments": {
    "FileName": "c:\\Development\\devinite\\OldMonolith\\devinite.Db\\DbLayer\\Classes\\Availability\\ProductAvailability\\ExpectedDateAtCustomerService.cs",
    "Line": 605,
    "Column": 13
  }
}
[dbug]: OmniSharp.Stdio.Host
        ************  Response ************ 
{
  "Request_seq": 220,
  "Command": "/v2/getcodeactions",
  "Running": true,
  "Success": true,
  "Message": null,
  "Body": {
    "CodeActions": [
      {
        "Identifier": "RR0094",
        "Name": "Remove braces"
      },
      {
        "Identifier": "RR0045",
        "Name": "Extract from containing statement"
      },
      {
        "Identifier": "RR0109",
        "Name": "Remove statement"
      },
      {
        "Identifier": "RR0033",
        "Name": "Duplicate statement"
      },
      {
        "Identifier": "RR0028",
        "Name": "Comment out statement"
      }
    ]
  },
  "Seq": 28552,
  "Type": "response"
}
[dbug]: OmniSharp.Stdio.Host
        ************ Request ************
{
  "Type": "request",
  "Seq": 221,
  "Command": "/findusages",
  "Arguments": {
    "FileName": "c:\\Development\\devinite\\OldMonolith\\devinite.Db\\DbLayer\\Classes\\Availability\\ProductAvailability\\ExpectedDateAtCustomerService.cs",
    "Line": 470,
    "Column": 45,
    "OnlyThisFile": false,
    "ExcludeDefinition": true
  }
}
[dbug]: OmniSharp.Stdio.Host
        ************  Response ************ 
{
  "Request_seq": 221,
  "Command": "/findusages",
  "Running": true,
  "Success": true,
  "Message": null,
  "Body": {
    "QuickFixes": [
      {
        "FileName": "c:\\Development\\devinite\\OldMonolith\\devinite.Db\\DbLayer\\Classes\\Availability\\ProductAvailability\\ExpectedDateAtCustomerService.cs",
        "Line": 133,
        "Column": 28,
        "EndLine": 133,
        "EndColumn": 40,
        "Text": "return ComputeStock(expectedDateAtCustomerParameters, estimatedDeliveryBySupplierResult, delayDueToLogistics);",
        "Projects": [
          "devinite.Db"
        ]
      }
    ]
  },
  "Seq": 28556,
  "Type": "response"
}
[dbug]: OmniSharp.Stdio.Host
        ************ Request ************
{
  "Type": "request",
  "Seq": 222,
  "Command": "/findusages",
  "Arguments": {
    "FileName": "c:\\Development\\devinite\\OldMonolith\\devinite.Db\\DbLayer\\Classes\\Availability\\ProductAvailability\\ExpectedDateAtCustomerService.cs",
    "Line": 454,
    "Column": 45,
    "OnlyThisFile": false,
    "ExcludeDefinition": true
  }
}
[dbug]: OmniSharp.Stdio.Host
        ************  Response ************ 
{
  "Request_seq": 222,
  "Command": "/findusages",
  "Running": true,
  "Success": true,
  "Message": null,
  "Body": {
    "QuickFixes": [
      {
        "FileName": "c:\\Development\\devinite\\OldMonolith\\devinite.Db\\DbLayer\\Classes\\Availability\\ProductAvailability\\ExpectedDateAtCustomerService.cs",
        "Line": 136,
        "Column": 28,
        "EndLine": 136,
        "EndColumn": 58,
        "Text": "return ComputeStockAtCentralWarehouse(expectedDateAtCustomerParameters, delayDueToLogistics);",
        "Projects": [
          "devinite.Db"
        ]
      }
    ]
  },
  "Seq": 28559,
  "Type": "response"
}

Process is terminated due to StackOverflowException.
```

**Comment by filipw** [comment]
thanks, but can you please post the full log?

**Comment by RononDex** [comment]
Where can I find that? the output console in vscode only shows the last X rows

**Comment by filipw** [comment]
it should be there, you can select everything with ctrl+a / cmd+a on a mac.

The output should start with something liket:

```
Starting OmniSharp server at 8/5/2019, 9:07:38 AM
    Target: ...... {path}

OmniSharp server started.
    Path: /Users/filipw/.vscode/extensions/ms-vscode.csharp-1.21.0/.omnisharp/1.34.0/run
    PID: 36349
```

**Comment by RononDex** [comment]
Its bit different with vim shortcuts :P but here it is
[Output.txt](https://github.com/OmniSharp/omnisharp-roslyn/files/3467353/Output.txt)
As you can see it does not start there, the output only shows the last 96k lines

**Comment by filipw** [comment]
resolved in #1579
